core/serv/src/main.rs

```diff
@@ -136,35 +141,56 @@ impl TryFrom<&CliArgs> for CliCtx {
     }
 }
 
+#[derive(Clone)]
 struct ServiceContext {
-    db: DbExecutor,
-    data_dir: PathBuf,
```

Comment:
This version of decentralized market still has data_dir. It shouldn't compile

Reply:
done